micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmGcMetrics.java

```diff
@@ -295,6 +311,13 @@ enum GcGenerationAge {
                 put("partial gc", YOUNG);
                 put("global garbage collect", OLD);
                 put("Epsilon", OLD);
+                // GPGC
+                put("GPGC New", YOUNG); // old naming
+                put("GPGC Old", OLD); // old naming
```

Comment:
Just for the sake of documentation:
It seems that the new names were introduced in Azul Prime 22.12.0.0, release date: 2022-12-19 so they are still supported, see the [release notes](https://docs.azul.com/prime/release-notes#prime_stream_22_12_0_0).